t/rose-date/00-basic.t

```diff
@@ -300,4 +300,13 @@ file_cmp "$TEST_KEY.err" "$TEST_KEY.err" <<'__ERR__'
 [FAIL] [UNDEFINED ENVIRONMENT VARIABLE] ROSE_TASK_CYCLE_TIME
 __ERR__
 #-------------------------------------------------------------------------------
+# Test ISO8601 duration convertion
+TEST_KEY=$TEST_KEY_BASE-duration
+run_pass "$TEST_KEY" rose date --as-total=s PT1M
+file_cmp "$TEST_KEY.err" "$TEST_KEY.err" </dev/null
+file_cmp "$TEST_KEY.out" "$TEST_KEY.out" <<'__OUT__'
+60.0
+__OUT__
+run_fail "$TEST_KEY.fail" rose date --as-total=y PT1M
```

Comment:
Should probably have tests for the expected failure.

Do you think we can have a few more test cases?